src/middlewared/middlewared/plugins/vm.py

```diff
@@ -320,6 +377,104 @@ def random_mac(self):
             return False
 
 
+class VMUtils(object):
+
+    def is_container(data):
```

Comment:
The only reason I have this check here is because specifically for this RancherOS image, I need to load it using grub-bhyve, however I intend to build an UEFI image in the future to be able to use VNC and so on.

In short, I don't want handle different bootloaders and focus more on UEFI because of the benefit of VNC usage. This method in specific, soon will be removed and we won't need to care if it is container or pure vm.